.github/workflows/ci.yaml

```diff
@@ -54,10 +60,66 @@ jobs:
       TRIAL_BRANCH: ${{ needs.metadata.outputs.TRIAL_BRANCH }}
     secrets: inherit
 
+  publish-mealkit:
+    needs: [metadata, amd64, arm64]
+    if: "!canceled()" && github.event_name == 'schedule' && needs.metadata.outputs.PUBLISH == 'true'
+    strategy:
+      fail-fast: true
```

Comment:
I can't think of a case where one of these fails and the others don't (maybe an API failure?). But what do you think about letting it be `false` so that we at least get some images published

Reply:
+1 for `fail-fast: false`